docs/getting_started/install.rst

```diff
@@ -5,6 +5,26 @@ Installation
 Basic Installation
 ------------------
 
+#. Make a directory (folder) for your project.
+#. Create a virtual environment. 
+    **Not sure how to create a virtual environment?**
+    Creating a virtual environment for your project only involves a few commands.
+    If you're on a Windows machine, you would run something like the following:
+
+    .. code-block:: console
+
+        $ python -m venv C:\Users\YourComputer\env\name_of_virtual_environment
```

Comment:
Can you add two code blocks, one for Windows one for macOS, Linux.

```rst
Windows (PowerShell):

.. code-block:: ps1con

    PS> python -m venv .\venv\
    PS> .\venv\Scripts\Activate.ps1

macOS, Linux:

.. code-block:: console

    $ python -m venv ./venv/
    $ source ./venv/bin/activate
```

Reply:
done